src/main/kotlin/data/model/Movie.kt

```diff
@@ -0,0 +1,53 @@
+package org.sessac.data.model
+
+import org.sessac.utils.ArtUtils
+import org.sessac.utils.FileUtils
+
+data class Movie(val title: String, val emoji: String, val seats: Array<Array<Seat>>) {
+    companion object {
+        fun createMovie(title: String, emoji: String, rows: Int, cols: Int): Movie {
+            val seats = Array(rows) { row -> Array(cols) { col -> Seat(row, col) } }
+            val movie = Movie(title, emoji, seats)
+            movie.loadSeatsFromFile()
+            return movie
+        }
+    }
+
+    fun displaySeats() {
+        seats.forEach { row ->
+            row.forEach { seat -> print("${seat.display()} ") }
+            println()
+        }
+    }
+
+    fun bookSeat(row: Int, col: Int): Boolean {
+        return if (row in seats.indices && col in seats[row].indices && seats[row][col].status == SeatStatus.AVAILABLE) {
+            seats[row][col].book()
+            saveSeatsToFile()
+            true
+        } else {
+            false
+        }
+    }
+
+    private fun saveSeatsToFile() {
+        val content = seats.map { row ->
+            row.joinToString(",") { it.status.name }
+        }
+        FileUtils.saveToFile("${title.replace(" ", "_")}_seats.txt", content)
+    }
```

Comment:
```suggestion
         FileUtils.saveToFile("${title.replace(" ", "_")}_seats.txt", content).use { writer ->
            content.forEach { line ->
                writer.write(line)
                writer.newLine()
            }
        }
    }
```
use를 사용하면 파일 입출력시에 메모리 누수를 방지 할 수 있다고 합니다!
use함수 써보시는 건 어떠신가요?